meshnets/modules/lightning_wrapper.py

```diff
@@ -23,6 +50,9 @@ def training_step(self, batch: Batch, _) -> dict:
         return {'loss': loss}
 
     def validation_step(self, batch: Batch, _) -> dict:
+
+        batch = self.normalize(batch)
```

Comment:
Why not avoid doing this again, by using the self.forward inside the loss computation?

Reply:
Same above.

Reply:
No real reason, but I guess I can indeed use the lightning module forward method instead yes.